src/main/kotlin/study/step2/Calculator.kt

```diff
@@ -0,0 +1,41 @@
+package study.step2
+
+import java.lang.IllegalArgumentException
+import java.util.Stack
+
+class Calculator {
+    fun calculate(input: String): Int {
+        val operandStack = Stack<Int>()
+        val operatorStack = Stack<Operator>()
+
+        val inputSplit = input.split(" ")
+
+        inputSplit.forEach { s: String ->
+            when {
+                s.toIntOrNull() != null && operandStack.isEmpty() -> operandStack.add(s.toInt())
+                Operator.values().map { o -> o.operator }.contains(s) -> addOperatorStack(operatorStack, s)
+                s.toIntOrNull() != null -> operandStack.add(process(operandStack, operatorStack, s.toInt()))
+                else -> throw IllegalArgumentException()
+            }
+        }
+
+        return operandStack.pop()
+    }
+
+    private fun addOperatorStack(operatorStack: Stack<Operator>, s: String) {
+        if (!operatorStack.empty()) {
```

Comment:
```suggestion
        if (operatorStack.isNotEmpty()) {
```
코틀린은 다양한 [확장함수](https://kotlinlang.org/docs/extensions.html)를 지원해요. 이를 찾아가는 것도 좋은 학습 방법 같습니다 😄